selectrum.el

```diff
@@ -1684,19 +1603,33 @@ suffix."
                                    suffix))))))
             res))))
 
-(defun selectrum--candidates-display-strings (candidates
-                                              highlighted-index
-                                              annot-fun
-                                              horizontalp)
+(defun selectrum--candidates-display-strings
+    (input current-index first-index-displayed ncands horizontalp)
   "Get display strings for CANDIDATES.
-HIGHLIGHTED-INDEX is the currently selected index. If ANNOT-FUN
-is non-nil don't add any annotations but call the function with
-the annotations of the currently highlighted candidate. If
-HORIZONTALP is non-nil candidates are supposed to be displayed
+INPUT is the input string for highlighting.
+CURRENT-INDEX is the index of the current candidate.
+FIRST-INDEX-DISPLAYED is the first index which is displayed.
+NCANDS is the maximum number of candidates to display.
+If HORIZONTALP is non-nil candidates are supposed to be displayed
 horizontally. TABLE defaults to `minibuffer-completion-table'.
 PRED defaults to `minibuffer-completion-predicate'. PROPS
 defaults to `completion-extra-properties'."
```

Comment:
These aren't valid anymore.

Reply:
I suspect some of the docstrings are not valid anymore. Makes sense to recheck them all.

Reply:
I fixed this and some more docstrings.